### PR TITLE
Update comments createLabel() thunk to more robustly handle source URLs

### PR DESCRIPTION
**Pull request by bvaughn**
Fixes the issue reported.

I'm pretty tired today though so please double check my logic.

### CODE REVIEW

File: src/ui/actions/comments.ts [hunk 1]
```diff
@@ -155,20 +155,23 @@ export function createLabels(
...
-      };
-    }
+    const { line, sourceUrl } = sourceLocation;
 
-    const { sourceUrl, line } = sourceLocation;
+    let sourceId: string | null = sourceLocation.sourceId || null;
 
     const filename = sourceUrl ? getFilenameFromURL(sourceUrl) : "unknown source";
+
     if (!sourceId) {
```
Comment (bvaughn) [inline src/ui/actions/comments.ts:165]: I'm not sure why we weren't already doing this check. Was there a good reason, or was this an oversight?

It seems analogous to this code that @jcmorrow recently changed:
https://github.com/replayio/devtools/blob/79d06aab168237d525f40c25d068f92ef7c16001/src/devtools/client/debugger/src/actions/sources/select.ts#L138-L142

Am I missing something?

Comment (markerikson) [reply src/ui/actions/comments.ts:165]: I haven't poked at this code myself, but the diff _seems_ valid...

**Comment by bvaughn** [comment]
Should we land this then?

**Comment by markerikson** [comment]
Sure, go ahead!

File: src/ui/actions/comments.ts (hunk 1 above)
Comment (jcmorrow) [reply src/ui/actions/comments.ts:165]: I'm not 100% sure I understand the change here, but it seems worth pointing out that even if we have an ID that doesn't mean it's the *correct* ID.

Comment (bvaughn) [reply src/ui/actions/comments.ts:165]: Thanks for that added context, Josh. I'll make that change before this lands then.

Comment (bvaughn) [reply src/ui/actions/comments.ts]: I think I've adjusted things now as discussed.

**Comment by bvaughn** [comment]
Huh. Lots of tests are failing (both new e2e tests and the old screenshot tests). I'm not sure if this is because the backend is being slow or what. I would not have expected this change to make things fail.

**Pull request merged**